src/notifications/notifications.ts

```diff
@@ -0,0 +1,131 @@
+import { SHOULD_TRACK_STORAGE_KEY } from '../options/privacy/constants'
+
+interface ButtonDefinition {
+    action?: any
+    label: string
+    primary?: boolean
+}
+
+interface NotifDefinition {
+    id: string
+    title: string
+    message: string
+    buttons?: ButtonDefinition[]
+}
+
+interface Notifs {
+    releaseTime: number
+    notifs: NotifDefinition[]
+}
+
+
+/**
+ * Add New Notification Here
+ * Add the new notification in {NOTIFS} array
+ * There are some fields that are mandtory and some of them are optional
+ * Fields and its description - * indicates that it is mandtory
```

Comment:
You should put these prop docs on the props themselves. To properly JDoc properties of an `interface` you can do something like this:

```ts
interface NotifDefinition {
    /** Should be unique (feature_name + notification + incNumber) */
    id: string
    ...
}
```
Multi-line is also fine.

I would also encourage having your `releaseTime` and your `notifs` array as separate exports. I don't see any reason for nesting them both within an object like this.

Reply:
JSDoc fix-ups look good. You should be able to remove this stuff now, as it's just duplicated